tests/lib/external/snapd-testing-tools/spread.yaml

```diff
@@ -35,7 +35,7 @@ backends:
                 storage: preserve-size
             - centos-9-64:
                 storage: preserve-size
-            - opensuse-15.4-64:
+            - opensuse-15.5-64:
```

Comment:
Should this be deleted outright, then 15.6 added when the time comes? I doubt we want a duplicate 15.5, unless this is intentionally a placeholder.

Reply:
thanks @olivercalder , seems to be a problem with the merge. I'll fix the change